typesense.gemspec

```
@@ -28,6 +28,6 @@ Gem::Specification.new do |spec|
 
   spec.add_dependency 'base64', '~> 0.2.0'
   spec.add_dependency 'faraday', '~> 2.8'
-  spec.add_dependency 'oj', '~> 3.16'
+  spec.add_dependency 'json', '~> 2.3'
```

I'm down to switching to native json, especially given the recent perf improvements. 

However, v2.3 seems to be [released](https://rubygems.org/gems/json/versions/2.3.1) in Jun 2020. The [last article](https://byroot.github.io/ruby/json/2025/01/14/optimizing-ruby-json-part-7.html) in this series says that the perf improvements were released in v2.9 in Dec 2024, unless it was back-ported? 

Should we update this to `~> 2.9`?

---

Yeah, should be fine.

```suggestion
  spec.add_dependency 'json', '~> 2.9'
```